content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -138,6 +138,15 @@ it is deleted.
 When you create the manifest for a Pod object, make sure the name specified is a valid
 [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
 
+### Pod OS
+
+You should set the `.spec.os.name` field to either `windows` or `linux` to indicate the OS on
+which you want the pod to run. These two are the only operating systems supported for now by 
+Kubernetes. In future, this list may be expanded. Please note that setting this field has no
+effect on scheduling of the pods. Setting the `.spec.os.name` helps to identify the pod OS
+authoratitively and is used for validation. Pod security standards also use this 
+field to relax certain policies for Windows pods.
```

Comment:
Is this right:
```suggestion
effect on {{< glossary_tooltip text="scheduling" term_id="kube-scheduler" >}} of the pods.
Setting the `.spec.os.name` helps to identify the pod OS
authoratitively and is used for validation. The kubelet refuses to run a Pod where you have
specified a Pod OS, if this isn't the same as the operating system for the node where
that kubelet is running.
The [Pod security standards](/docs/concepts/security/pod-security-standards/) also use this
field to avoid enforcing policies that aren't relevant to that operating system.
```

Reply:
Both kubelet and pod api validation use this field. So. you're right.